CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # `ipyradiant` CHANGELOG
 
+## 0.1.2 (J_e) (2021-01-21)
+
+- adds `RDF2NX` transformer for converting RDF to a networkx labelled property graph.
```

Comment:
in the future, links to issues (or PRs) better

Reply:
I will track these better throughout the release cycle in the future.